devel/lerna/Portfile

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        lerna lerna 3.4.3 v
+homepage            https://lernajs.io/
+
+platforms           darwin
+categories          devel
+license             MIT
+installs_libs       no
+
+maintainers         {gmail.com:herby.gillot @herbygillot} openmaintainer
+
+description         A tool for managing JavaScript projects with multiple \
+                    packages.
+
+long_description    Splitting up large codebases into separate independently \
+                    versioned packages is extremely useful for code sharing. \
+                    However, making changes across many repositories is messy \
+                    and difficult to track, and testing across repositories \
+                    gets complicated really fast.  \
+                    Lerna is a tool that optimizes the workflow around \
+                    managing multi-package repositories with git and npm. 
+
+depends_run         port:nodejs10,port:npm6
+depends_build       port:npm6
+
+checksums           rmd160  e124c044e6cabbc9c372d16630ff52a17297f05d \
+                    sha256  0be29ed6f713ab331492348a46c8d63221d85d1ef2873d5af9e110ef105aa3ad \
+                    size    377699
+
+use_configure       no
+
+build.cmd           npm
```

Comment:
Use `${prefix}/bin/npm` to avoid problems for users who may have customized `binpath` in macports.conf.